libraries/AP_Mission/AP_Mission.cpp

```diff
@@ -477,6 +477,73 @@ bool AP_Mission::set_current_cmd(uint16_t index)
     return true;
 }
 
+// returns false on any issue at all.
+bool AP_Mission::set_item(uint16_t index, mavlink_mission_item_int_t& src_packet)
+{
+        // this is the on-storage format
```

Comment:
Looks like the indenting is 8spaces instead of 4?

Reply:
ran thru astyle, fixed.